src/w3c/abstract.js

```diff
@@ -30,18 +31,33 @@ const localizationStrings = {
 const l10n = getIntlData(localizationStrings);
 
 export async function run() {
-  const abs = document.getElementById("abstract");
-  if (!abs) {
-    const msg = 'Document must have one element with `id="abstract"`.';
+  const abstract = findAbstract();
+  if (!abstract) {
+    const msg = 'Document must have one `<section id="abstract">` in the body.';
```

Comment:
Nit: "in the body" seems a bit redundant, no?
```suggestion
    const msg = 'Document must have one `<section id="abstract">`.';
```